### PR TITLE
Moving DataDog tracing to a central location

### PR DESCRIPTION
**Pull request by pshomov**
# \<Description\>

Adding APM support for the NextJS services as well

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by gudjong — DISMISSED** [review]

**Review by sindrig — APPROVED** [review]

**Review by jeremybarbet — APPROVED** [review]

File: libs/infra-next-server/src/lib/bootstrap.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import '@island.is/infra-tracing'
```
Comment (darrikonn) [inline libs/infra-next-server/src/lib/bootstrap.ts:1]: Is this supposed to be the nest-server?

Comment (darrikonn) [reply libs/infra-next-server/src/lib/bootstrap.ts:1]: (since you added this to the next-server)

Comment (pshomov) [reply libs/infra-next-server/src/lib/bootstrap.ts:1]: 🤦🏻
It is supposed to be in both.

Comment (pshomov) [reply libs/infra-next-server/src/lib/bootstrap.ts:1]: Fixed.

**Review by fjandakornid — APPROVED** [review]

**Review by Thorrik58 — DISMISSED** [review]

**Review by gudjong — APPROVED** [review]
LGTM

**Review by Ingimarsson — APPROVED** [review]

**Review by eirikurn — APPROVED** [review]

**Review by stjanilofts — APPROVED** [review]

**Pull request merged**